.github/workflows/main.yml

```diff
@@ -87,9 +85,33 @@ jobs:
           EMBER_DATA_FEATURE_OVERRIDE: DISABLE_ALL
         run: yarn test
 
+  fastboot-tests:
+    needs: [lint]
+    timeout-minutes: 6
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        scenario: [fastboot-with-ember-fetch, fastboot-with-jquery]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - if: matrix.os == 'macos-latest'
```

Comment:
Is there a way to factor this setup set as we are repeating it in a bunch of places

Reply:
no, each of these is a bit different or a unique scenario and abstracting to a stand-alone action would result in just as many lines